src/main/java/com/fireteam322/frc/robot/subsystems/HookClimbingSubSystem.java

```diff
@@ -0,0 +1,24 @@
+package com.fireteam322.frc.robot.subsystems;
+
+import com.fireteam322.frc.robot.Constants;
+
+import edu.wpi.first.wpilibj.motorcontrol.MotorController;
+import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class HookClimbingSubSystem extends SubsystemBase {

```

Comment:
I made the subsystem for the hooks here @Alijah2424

Reply:
We would have to add the stop , and run and any other functions , I made build and interface for the most common